.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
```suggestion
<!-- Please delete options that are not relevant. --> 
```
Optional little thing, you can use this type of html comment notation in order to make the instructions to the person who is filling out the template appear to them but not in the final rendered markdown.